### PR TITLE
[bitnami/grafana] Add Ingress extra paths to prepend to every host configuration (#2295)

### PR DESCRIPTION
**Pull request by evgmoskalenko**
Add Ingress extra paths to prepend to every host configuration. Useful when configuring [custom actions with AWS ALB Ingress Controller](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/ingress/annotation/#actions).

This helps manage to route the paths using ingress controllers. For example - alb-ingress-controller and chart ingress settings:

```
ingress:
  enabled: true
  annotations:
    kubernetes.io/ingress.class: "alb"
    alb.ingress.kubernetes.io/scheme: "internet-facing"
    alb.ingress.kubernetes.io/backend-protocol: "HTTP"
    alb.ingress.kubernetes.io/ip-address-type: "ipv4"
    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
    # Redirect rules - actions.ssl-redirect
    alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'
```

And ingress path settings:

```
ingress:
  enabled: true
  ...
  hosts:
    - name: ${release_fqdn}
      paths: ["/*"]
      extraPaths:
        - path: /*
          backend:
            serviceName: ssl-redirect
            servicePort: use-annotation
```

In this case, all requests from port 80 will be redirected to 443 port. More info you can find [here](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/ingress/annotation/#actions).

**Checklist**
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/grafana/templates/ingress.yaml [hunk 1]
```diff
@@ -17,12 +17,18 @@ spec:
...
         {{- $paths := default (list "/") .paths }}
         {{- range $paths }}
-          - path: {{ . }}
-            backend:
-              serviceName: {{ include "grafana.fullname" $ }}
-              servicePort: http
+        - path: {{ . }}
+          backend:
+            serviceName: {{ include "grafana.fullname" $ }}
+            servicePort: http
```
Comment (dani8art) [inline bitnami/grafana/templates/ingress.yaml:31]: Please indent this array. 
```yaml
array:
  - item1
  - item2
```

Comment (evgmoskalenko) [reply bitnami/grafana/templates/ingress.yaml]: Hi @dani8art 

I create commit. Thanks.. :-)

**Comment by evgmoskalenko** [comment]
> Please, resolve `Chart.yaml` conflicts.

Resolved

**Comment by dani8art** [comment]
LGTM! Waiting for CI/CD.

**Pull request merged**